custom/icds_reports/management/commands/sql_scripts/ap_data_pull_1.sql

```diff
@@ -0,0 +1,28 @@
+COPY(SELECT
```

Comment:
For the simplification of the management command. i think you can combine all the SQL queries into a single SQL file. I was trying to split them into different files so that if we were able to write the simple one early, could  have been run at night or in the morning. Itself. But I think its better now to just merge different csv files.